test/e2e/autoscale_test.go

```diff
@@ -357,12 +385,152 @@ func assertAutoscaleUpToNumPods(ctx *testContext, curPods, targetPods float64, d
 	}
 }
 
+func getUniqueHostPerConnection(t *testing.T, ctx *testContext, size int) (map[string]*websocket.Conn, error) {
```

Comment:
This should go to a different file, I think.
Also the name is a bit off... `getUniqueHostConnections` ? or just `uniqueHostConnections`, which is more gostyle.

Reply:
`uniqueHostConnections` sounds good.

Reply:
also moving it to e2e.go